domain/src/test/kotlin/no/nav/su/se/bakover/domain/søknadsbehandling/StatusovergangTest.kt

```diff
@@ -779,6 +786,7 @@ internal class StatusovergangTest {
                 iverksattAvslagBeregning,
                 iverksattAvslagVilkår,
                 iverksattInnvilget,
+                søknadsbehandlingLukket().second
```

Comment:
Ikke verdens undergang at _all_ testdata ikke henger sammen, men kunne evt vurdert å brukt `opprettet.lukkSøknadsbehandling()` for å hekte deg på de allerede sammenhengende dataene i denne filen.